movie_REST_API/MovieUnitTests/Mock/MockNetworkService.swift

```diff
@@ -0,0 +1,46 @@
+// MockNetworkService.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import XCTest
+
+@testable import movie_REST_API
+
+/// Mock NetworkService
```

Comment:
```suggestion
/// Mock NetworkService
```
Дока на русском плиз